### ISSUE TITLE
Underground Biomes Constructs and ore textures

### ISSUE THREAD

**Issue by Thunderpriest** [Blood-Asp/GT5-Unofficial#172]
Faced a problem while playing with GT5 Unofficial and UBC: UBC replaces vanilla stone with its own and GT sets the ore background textures to netherrack. It would probably be better if it set at least the vanilla stone textures.
![2015-09-09_18 25 19](https://cloud.githubusercontent.com/assets/13168286/9767816/146fd488-5731-11e5-901c-2562963a49e0.png)
![2015-09-09_18 28 21](https://cloud.githubusercontent.com/assets/13168286/9767815/146f1804-5731-11e5-8a73-757e3ed78058.png)
![2015-09-09_18 29 17](https://cloud.githubusercontent.com/assets/13168286/9767817/147039b4-5731-11e5-90f1-a6b42bf36d1c.png)
![2015-09-09_18 30 00](https://cloud.githubusercontent.com/assets/13168286/9767814/146e2ad4-5731-11e5-9299-ecead26326d3.png)

**Comment by Blood-Asp** [comment]
What UBC version did you use? The last one i used worked just fine and made stone textures.

**Comment by Thunderpriest** [comment]
1.0 for MC 1.7.2, as it says. Not sure about is it 0.6 or 0.7 from here:
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1293503-underground-biomes-constructs

I've also tried some other version, usure which exactly.
I'll try some more other versions soon [didn't actually have much time for that =D], just notified in case it is useful to support compatibility with that version.

I can send you the .jar file of the UBC I used.
And in case it's a single version bug - ... Well, I'm stupid then)
